src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier;
+
+    public Lottery(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        Color ballColor = Color.valueOf(colorSupplier.getRandomColor());
+        int ballNumber = random.nextInt(100);
```

Comment:
create a constant from 100
For example: private static final String MY_CONSTANT = "hello";